spec/lib/api/2.0/skus-spec.js

```diff
@@ -0,0 +1,360 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Skus', function () {
+    var workflowApiService;
+    var _;
+    before('start HTTP server', function () {
+        this.timeout(5000);
+        workflowApiService = {
+            createAndRunGraph: sinon.stub()
+        };
+        helper.setupInjector([
+            helper.require("/lib/services/sku-pack-service"),
+            dihelper.simpleWrapper(function() { arguments[1](); }, 'rimraf')
+        ]);
+        return helper.startServer([
+            dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
+        ]).then(function() {
+            _ = helper.injector.get('_');
+        });
+    });
+
+    beforeEach('reset test DB', function () {
+        return helper.reset();
+    });
+
+    after('stop HTTP server', function () {
+        return helper.stopServer();
+    });
+
+    it('should return an empty array from GET /skus', function () {
+        return helper.request().get('/api/2.0/skus')
+            .expect('Content-Type', /^application\/json/)
+            .expect(200, []);
+    });
+
+    describe('created SKU', function () {
+        var input;
+        var sku;
+        var node;
+
+        beforeEach('create node', function () {
+            var waterline = helper.injector.get('Services.Waterline');
+            return waterline.nodes.create({ name: 'sku test node' }).then(function (node_) {
```

Comment:
Hi Felix,
My reply goes for all three cases that you pointed out regarding the database access.
I agree with that the unit test should not be accessing or depending on the on the db, but this is an old code from the API 1.1. I just reused for the 2.0.

If you want, I can create a story to refactor or rewrite all of the tests in here in order to make them indepentdent, but this would be outside the scope of this story